sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/UnknownIdentifier.java

```diff
@@ -43,11 +45,11 @@ public boolean equals(Object that) {
         }
 
         UnknownIdentifier thatId = (UnknownIdentifier) that;
-        return this.id.equals(thatId.id);
+        return this.getRawId().equals(thatId.getRawId());
     }
 
     @Override
     public int hashCode() {
-        return id.hashCode();
+        return getRawId().hashCode();
     }
```

Comment:
Can we get rid of these overriding members as they contain the same functionality as the base?